### PR TITLE
Update some packets to 1.15.2

### PR DESCRIPTION
**Pull request by ImNowAPro**

### CODE REVIEW

File: PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerNamedEntitySpawn.java [hunk 1]
```diff
@@ -181,24 +181,4 @@ public float getPitch() {
 	public void setPitch(float value) {
 		handle.getBytes().write(1, (byte) (value * 256.0F / 360.0F));
 	}
-
```
Comment (dmulloy2) [inline PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerNamedEntitySpawn.java:184]: Do these need to be removed? They can be kept for legacy compatibility

File: PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerRelEntityMove.java [hunk 2]
```diff
@@ -78,28 +78,28 @@ public Entity getEntity(PacketEvent event) {
 		return getEntity(event.getPlayer().getWorld());
 	}
 
-	public int getDx() {
```
Comment (dmulloy2) [inline PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerRelEntityMove.java:81]: Might be worth keeping these as returning ints (not like you can do stuff with shorts you couldn't do with ints) and doing a version check to see if you should read from Integers or Shorts

File: PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerRelEntityMoveLook.java [hunk 3]
```diff
@@ -83,53 +83,53 @@ public Entity getEntity(PacketEvent event) {
 	 * 
 	 * @return The current DX
 	 */
-	public double getDx() {
```
Comment (dmulloy2) [inline PacketWrapper/src/main/java/com/comphenix/packetwrapper/WrapperPlayServerRelEntityMoveLook.java:86]: Is the scaling done internally now? Or will this cause side effects?

**Comment by dmulloy2** [comment]
Superseded by https://github.com/dmulloy2/PacketWrapper/commit/5262e36c72b66a6b94eb97edc710094329a3ce5e

**Comment by MiniDigger** [comment]
@dmulloy2 where these generated by wrapit?

**Pull request closed without merging**